src/platform-includes/distributed-tracing/custom-instrumentation/dotnet.mdx

```diff
@@ -0,0 +1,64 @@
+On this page you will learn how to manually propagate trace information into and out of your .NET application. Please note, that you do not need to do this manually, if you [use one of our supported frameworks](/platforms/dotnet/usage/distributed-tracing/#how-to-use-distributed-tracing), or you have our <PlatformLink to="/performance/">performance monitoring feature</PlatformLink> turned on.
+
+To set it up manually, all you have to do is to make sure your application extracts incoming headers and to set those headers again when making an outgoing request within your application.
+
+## Step 1) Extract Incoming Tracing Information
+
+Incoming tracing information has to be extracted and stored in memory for later use. Sentry provides the `ContinueTrace()` function to help you with this. Incoming tracing information can come from HTTP headers, for example, by another Sentry SDK used in your frontend project.
+
+Here's an example of how to extract and store incoming tracing information using `ContinueTrace()`:
+
+```csharp
+using System;
+using System.Net.Http;
+using Sentry;
+
+var httpContext = ...;
+
+var sentryTraceHeader = httpContext.Request.Headers.GetValueOrDefault("sentry-trace");
+var sentryBaggageHeader = httpContext.Request.Headers.GetValueOrDefault("baggage");
+
+try
+{
+    var transactionContext = SentrySdk.ContinueTrace(sentryTraceHeader, sentryBaggageHeader);
+    var transaction = Sentry.startTransaction(transactionContext);
+}
+catch (FormatException e)
+{
+    // Handle invalid trace header
+    e.Data["bad-sentry-header"] = sentryTraceHeader;
+    SentrySdk.CaptureException(e);
+}
+```
+
+If you pass these headers to Sentry's `ContinueTrace()` function it will store them in memory for later use.
+
+## Step 2) Inject Tracing Information to Outgoing Requests
+
+For distributed tracing to work, the two headers `sentry-trace` and `baggage`, must now also be added to outgoing requests.
+
+If you are sending outgoing HTTP requests with a HTTP client that has the `SentryHttpMessageHandler` this tracing information is automatically added to outgoing requests. You do not have to enable Performance for distributed tracing to work.
```

Comment:
It feels like there's a word missing here? Or I might be reading it wrong.

Here's a potential suggestion that sounds more grammatically correct, but please make sure it's factually correct :D

```suggestion
If you send outgoing HTTP requests with an HTTP client that uses `SentryHttpMessageHandler`, this tracing information is automatically added to outgoing requests. You do not have to enable Performance for distributed tracing to work.
```